Source/pages/knowMore.html

```diff
@@ -405,70 +419,162 @@ <h4 class="panel-title">
+                        <p class="item-intro text-muted">Trees absorption rate & facts.</p>
+                        <p>
+                            <div>
+                                <form id="absorptionForm" class="form-main" role="form">
+                                    <h3>Aborption Time Calculator</h3>
+                                    <div class="form-group">
+                                        <label>Unit System</label><br>
+                                        <label class="radio-inline">
+                                            <input type="radio" name="unitSystem" value="uscustomary" checked>US customary
+                                        </label>
+                                        <label class="radio-inline">
+                                            <input type="radio" name="unitSystem" value="metric" >Metric
+                                        </label>
+                                        <label class="radio-inline">
+                                            <input type="radio" name="unitSystem" value="imperial" >Imperial
+                                        </label>
+                                    </div>
+                                    <div class="form-group">
+                                        <label for="treeType">Tree</label><br>
+                                        <select id="treeType" name="treeType" class="form-control">
+                                        <!-- <option value="uscustomary">lbs</option> -->
+                                        </select>
+                                    </div>
+                                    <div class="form-group">
+                                        <label for="usrEmission">CO<sub>2</sub> Emitted</label><br>
+                                        <input type="number" class="form-control form-value" id="usrEmission" name="emission" value="1.2">
+                                        <select class="form-control form-unit" disabled="" value="uscustomary" id="emissionUnit">
+                                            <option value="uscustomary">lbs</option>
+                                            <option value="imperial">lbs</option>
+                                            <option value="metric">kg</option>
+                                        </select>
+                                    </div>
+                                    <div id="absorptionFormSubmit" class="btn btn-md form-submit">Submit</div>
+                                    <div id="outputAbsorption" class="outputDiv"></div>
+
+                                </form>
+                            </div>
+                        </p><br>
+                        <h3 class="text-center">Facts</h3>
+                        <p class="text-left">
+                        <!-- http://forestlearning.edu.au/images/resources/How%20carbon%20is%20stored%20in%20trees%20and%20wood%20products.pdf -->
```

Comment:
This material is copyrighted. So, we can't simply copy-paste it here. We must either write our own paragraphs, or contact the authors of the material and ask for their permission to reproduce it here.